src/components/Select/SelectOption.vue

```diff
@@ -1,5 +1,29 @@
 <template>
-  <div
+  <Tooltip 
```

Comment:
I'm not 100% happy with this, there is some code duplication, but i couldn't came up with better solution. I've consulted this with Mateusz Zieliński and he also said in that case duplicating some code would be neccesary. If you have any suggestions to make it better, please write that down